content/applications/inventory_and_mrp/maintenance/maintenance_reporting.rst

```diff
@@ -0,0 +1,295 @@
+- :guilabel:`Expand all`: all the available rows and columns of the pivot data table expand fully.
+- :guilabel:`Download xlsx`: the pivot data table is downloaded as an .xlsx file.
+
+Kanban view
+-----------
+
+Click the :icon:`oi-view-kanban` :guilabel:`(Kanban view)` icon to change to Kanban view.
+
+With the :guilabel:`Kanban` view, all open maintenance requests are displayed in Kanban-style
+columns, in their respective stages of the maintenance process.
+
+Each maintenance request appears on its own task card, and each task card can be dragged-and-dropped
+to a different stage of the Kanban pipeline.
+
+Each column has a name (i.e. :guilabel:`In Progress`). Hovering at the top of a column reveals a
+:icon:`fa-cog` :guilabel:`(cog)` icon. Clicking the :icon:`fa-cog` :guilabel:`(cog)` icon reveals a
+list of options for that column: :guilabel:`Fold`, :guilabel:`Edit`, :guilabel:`Automations`, and
+:guilabel:`Delete`.
+
+.. image:: maintenance_reporting/maintenance-reporting-kanban-columns.png
+   :align: center
+   :alt: Column options for stage in Kanban view.
+
+Clicking :guilabel:`Fold` folds the column to hide its contents.
+
+Clicking :guilabel:`Edit` opens an :guilabel:`Edit: (stage name)` pop-up window, with the
+corresponding stage name, wherein the column's details can be edited. The following are the column
+options that can be edited:
+
+.. image:: maintenance_reporting/maintenance-reporting-stage-edit-popup.png
+   :align: center
+   :alt: Edit In Progress pop-up window.
+
+- :guilabel:`Name`: the name of the stage in the Kanban pipeline.
+- :guilabel:`Folded in Maintenance Pipe`: when checked, this stage's column is folded by default in
+  the :guilabel:`Kanban` view type.
+- :guilabel:`Request Confirmed`: when this box is not ticked, and the maintenance request type is
+  set to *Work Center*, no leave is created for the respective work center when a maintenance
+  request is created. If the box *is* ticked, the work center is automatically blocked for the
+  listed duration, either at the specified date, or as soon as possible, if the work center is
+  unavailable.
+- :guilabel:`Sequence`: the order in the maintenance process, in which this stage appears.
+- :guilabel:`Request Done`: if ticked, this box indicates this stage is the final step of the
+  maintenance process. Requests moved to this stage are closed.
+
+Once ready, click :guilabel:`Save & Close`.
+
+List view
+---------
+
+Click the :icon:`oi-view-list` :guilabel:`(list view)` icon to change to list view.
+
+With the :guilabel:`List` view selected, all open maintenance requests are displayed in a list, with
+information about each request listed in its respective row.
+
+The columns of information displayed in this view type are the following:
+
+- :guilabel:`Subjects`: the name assigned to the maintenance request.
+- :guilabel:`Request Date`: the date on which the maintenance request was created.
```

Comment:
This does not appear in my runbot- maybe check to see if this is a default item? It didn't even show up under the additional items icon, so I am not sure if this is abailable? I am in a 17.0 runbot though.

Reply:
I'm not seeing it now either, guessing it was a custom field added. Suggestion implemented ✅